apple-pie/index.html

```diff
@@ -5,17 +5,51 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
     <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+    <header>
+    <h1>Apple Pie</h1>
+    </header>
+    <div>
```

Comment:
A `<div>` or a `<section>` do the exact same thing, it's just a matter of semantics/good practices, and it is important for people that use screen readers. 
So there is no need to open a `<div>` tag every time you have a `<section>` you can just use the `<section>`:

This:
```
    <div>
      <section>
        <h4>Ingredients</h4>
        <ul>
          <li>1 recipe pastry for a 9 inch double crust pie</li>
          <li>1/2 cup unsalted butter</li>
          <li>3 tablespoons all-purpose flour</li>
          <li>1/4 cup water</li>
          <li>3 tablespoons all-purpose flour</li>
          <li>1/4 cup water</li>
          <li>1/2 cup white sugar</li>
          <li>1/2 cup packed brown sugar</li>
          <li>8 Granny Smith apples - peeled, cored and sliced</li>
        </ul>
        <img src="images/recipe-info.png" alt="information about the recipe" class="recipeImage"/>
      </section>
    </div>
```

Is the same as this:
```
      <section>
        <h4>Ingredients</h4>
        <ul>
          <li>1 recipe pastry for a 9 inch double crust pie</li>
          <li>1/2 cup unsalted butter</li>
          <li>3 tablespoons all-purpose flour</li>
          <li>1/4 cup water</li>
          <li>3 tablespoons all-purpose flour</li>
          <li>1/4 cup water</li>
          <li>1/2 cup white sugar</li>
          <li>1/2 cup packed brown sugar</li>
          <li>8 Granny Smith apples - peeled, cored and sliced</li>
        </ul>
        <img src="images/recipe-info.png" alt="information about the recipe" class="recipeImage"/>
      </section>
```